### PR TITLE
Added Privacy-Policy

### PR DESCRIPTION
**Pull request by Sarthakkashyapp**
Added Privacy Policy keeping it clean and minimal design which is responsive as a privacy page should be, also added privacy policy to the footer so that user can directly jump onto it.(Also there were some conflicts in Navbar.jsx also fixed that from my end).

Issue resolved : #183 

@AlfiyaSiddique again Thankyou so much for the opportunity and always giving feedbacks. It was great working on this project.😊. Also add the labels **hacktoberfest-accepted**, **gssoc-ext** and **level2** or **level3**. Thanks you 🙌

https://github.com/user-attachments/assets/e083df8d-0625-46c9-b959-8a3b32177167

### CODE REVIEW

File: frontend/src/Components/PrivacyPolicy.jsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+      <section className="mb-6">
+        <h2 className="text-2xl font-semibold mb-2">6. Contact Us</h2>
+        <p>
+          If you have any questions or concerns about this Privacy Policy or our data practices, please contact us:
+        </p>
+        <ul className="list-disc list-inside">
+          <li><strong>Email:</strong> <a href="mailto:privacy@tastytrails.com" className="text-blue-500 underline">privacy@tastytrails.com</a></li>
+          <li><strong>Address:</strong> [Your Company Address]</li>
+        </ul>
```
Comment (AlfiyaSiddique) [inline frontend/src/Components/PrivacyPolicy.jsx:68]: Remove the address line, not requires rest look good!

**Comment by AlfiyaSiddique** [comment]
@Sarthakkashyapp looks good!
Please update your PR with the latest commit and fix the one improvement in the code in above comment.
Then this PR is good to go

File: frontend/src/Components/PrivacyPolicy.jsx (hunk 1 above)
Comment (Sarthakkashyapp) [reply frontend/src/Components/PrivacyPolicy.jsx:67]: Removed the address line

**Comment by Sarthakkashyapp** [comment]
@AlfiyaSiddique Made the changes (removed the address line), commited the changes , also this branch also showing already up-to-date on running git pull origin master.

**Comment by AlfiyaSiddique** [comment]
@Sarthakkashyapp please try to resolve conflicts this time. Last time I resolved the conflicts of your PR because you were not able to.
Please learn it from youtube and try once

**Comment by Sarthakkashyapp** [comment]
> @Sarthakkashyapp please try to resolve conflicts this time. Last time I resolved the conflicts of your PR because you were not able to. Please learn it from youtube and try once

@AlfiyaSiddique I tried infact I've resolved conflicts it earlier but i dont know why everytime I  am getting already up to date after running git pull origin master usually it shows conflicting files in source control panel so sorry to disturb you like this I'm so sorry literally.

**Comment by Sarthakkashyapp** [comment]
@AlfiyaSiddique if possible guide me any other way how can I resolve I'll learn something from it Thankyou 😊

**Comment by Sarthakkashyapp** [comment]
@AlfiyaSiddique resolved conflict issues kindly look into this. Thankyou.

**Review by AlfiyaSiddique — APPROVED** [review]

**Comment by AlfiyaSiddique** [comment]
Good work @Sarthakkashyapp

**Comment by Sarthakkashyapp** [comment]
> Good work @Sarthakkashyapp 

Thank you so much @AlfiyaSiddique for always guiding me ☺️

**Pull request merged**